.editorconfig

```
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
```

This line about final newlines needs a newline.
```suggestion
insert_final_newline = true
```

---

...which makes the rest of the "nits" here confusing...just FYI...